product_docs/docs/pgd/4/rel_notes/pgd_4.3.1+p1_rel_notes.mdx

```diff
@@ -1,9 +1,9 @@
 ---
-title: "Release notes for EDB Postgres Distributed version 4.3.1-1"
-navTitle: "Version 4.3.1-1"
+title: "Release notes for EDB Postgres Distributed version 4.3.1+p1"
+navTitle: "Version 4.3.1+p1"
 ---
 
-EDB Postgres Distributed version 4.3.1-1 is a minor release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
+EDB Postgres Distributed version 4.3.1+p1 is a minor release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
```

Comment:
So is it "patch release" or "minor release"? What's the purpose of this `+p1`suffix? Do we explain this somewhere?

Reply:
Thanks for clarifying the meaning of the version suffix, @djw-m - for consistency, I'd like to propose we stick to calling these updates "patch releases" (abbreviated via the "p" suffix).

Reply:
Semver says the third digit of the version number is the patch release number which is how we got here. @mwanner - want to express a preference on what we call these patch/rebundled releases?

Reply:
I don't mind us calling this a patch release as well.